src/commands/mod.rs

```diff
@@ -515,6 +517,70 @@ struct NewArgs {
     insert_before: bool,
 }
 
+/// Move the current working copy commit to the next child revision in the
+/// repository. The command moves you to the next child in a linear fashion.
```

Comment:
Keep the first sentence for `jj help` and move the second sentence further below, so it shows up for `jj next --help`